lib/commands/element.js

```diff
@@ -249,7 +249,26 @@ const commands = {
 
     const atomsElement = this.getAtomsElement(el);
     await this.executeAtom('click', [atomsElement]);
+
+    if (this.opts.sendKeyStrategy !== 'oneByOne') {
+      await this.setValueWithWebAtom(atomsElement, value);
+      return;
+    }
+    for (const char of prepareInputValue(value)) {
+      await this.setValueWithWebAtom(atomsElement, char);
+    }
+  },
+
+  /**
+   * Set value with Atom for Web. This method calls `type` atom only.
+   * Expected to be called as part of {@linkcode setValue}.
+   * @this {XCUITestDriver}
+   * @param {import('./types').AtomsElement<string>} atomsElement A target element to type the given value.
+   * @param {string|string[]|number} value The actual text to type.
```

Comment:
I am not quite sure we'd like to send numbers here. It would be safer to convert the input to a string explicitly

Reply:
Sure. just I set acceptable values but agree with it for safe